### DISCUSSION TITLE
Supported @docusaurus/theme-live-codeblock syntax

### DISCUSSION THREAD

**Discussion by filiptammergard** [facebook/docusaurus#8128] [Q&A]
Hey!

I'm using `@docusaurus/theme-live-codeblock` in my blog. Noticed that the nullish coalescing operator (`??`) isn't working:

![image](https://user-images.githubusercontent.com/44197016/192120693-e66738af-0c5f-4c0c-a557-746db6987553.png)

You can try it out quickly in the interactive code editor in the docs: https://docusaurus.io/docs/markdown-features/code-blocks#interactive-code-editor

The exact same code works fine in the `react-live` playground:  https://react-live.netlify.app/

![image](https://user-images.githubusercontent.com/44197016/192120699-f9c10fc6-40dd-465e-aae8-cc131e914dc6.png)

Is this intended? Is it possible to change something to support `??`?

**Comment by slorber** [comment]
We are on an older version of react-live (2.2.3) that uses bublé to transpile. Newer versions 3.x use Sucrase.

Also there's this PR on which we should make a choice: https://github.com/facebook/docusaurus/pull/6589

**Reply by filiptammergard** [reply]
Is there something I can help out with to make live codeblocks work with newer syntax @slorber?
